readme.md

```diff
@@ -1,14 +1,34 @@
-# Search plugin for [Refinery CMS](http://www.refinerycms.com)
+# Refinery CMS Search [![Build Status](https://travis-ci.org/refinery/refinerycms-search.svg?branch=master)](https://travis-ci.org/refinery/refinerycms-search)
+
+Simple search engine for [Refinery CMS](http://refinerycms.com). By default, it supports search in `Refinery::Page` and `Refinery::Blog::Post` (if installed), it can be easily configured for another Rails models.
+
+This version of `refinerycms-search` supports Refinery 3.x and Rails 4.1.x.  To use Rails 3.x / Refinery 2.1.x use the [refinerycms-search "Refinery CMS 2-1 stable branch"](http://github.com/refinery/refinerycms-search/tree/2-1-stable).
 
 Powered by: [acts_as_indexed](https://github.com/dougal/acts_as_indexed) and [refinerycms-acts-as-indexed](https://github.com/refinery/refinerycms-acts-as-indexed) -
 Check his readme and documentation for more info on how it works.
 
+
+### In summary you can
+
+* Search in `Refinery::Page` and `Refinery::Blog::Post` models ;
+* Add others models in your configuration ;
+* Scope search query with a special method ;
+* Use a custom search page url ;
```

Comment:
URL is an acronym so should be `URL`, not `url`.
I'd personally remove the semicolons too: as it's already a list.